include/boost/url/any_allocator.hpp

```diff
@@ -0,0 +1,126 @@
+//
+// Copyright (c) 2022 Alan Freitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/CPPAlliance/url
+//
+
+#ifndef BOOST_URL_ANY_ALLOCATOR_HPP
+#define BOOST_URL_ANY_ALLOCATOR_HPP
+
+#include <boost/url/detail/config.hpp>
+#include <boost/url/detail/any_allocator.hpp>
+
+#include <memory>
+#include <type_traits>
+
+namespace boost {
+namespace urls {
+
+/** A type-erased allocator with shared ownership.
+
+    This type satisfies the requirements for
+   <em>Allocator</em>
+
+    @par Specification
+    @li <a
+     href="https://en.cppreference.com/w/cpp/named_req/Allocator"
+       >Allocator (cppreference.com)</a>
```

Comment:
Sometimes you have to break the rules. In this case I would format it thusly:
```
    @par Specification
    @li <a href="https://en.cppreference.com/w/cpp/named_req/Allocator"
        >Allocator (cppreference.com)</a>
```

Reply:
Even if it goes on "forever"? That's more than 100 chars.